tests/foreman/ui/test_repository.py

```diff
@@ -740,6 +742,41 @@ def test_positive_end_to_end_custom_ostree_crud(session, module_prod):
         assert not session.repository.search(module_prod.name, new_repo_name)
 
 
+def test_positive_sync_ansible_collection_gallaxy_repo(session, module_prod):
+    """Sync ansible collection repository from ansible gallaxy
+
+    :id: f3212dbd-3b8a-49ad-ba03-58f059150c04
+
+    :expectedresults: All content synced successfully
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+
+    """
+    repo_name = f'gallaxy-{gen_string("alpha")}'
+    requirements = '''
```

Comment:
Note: You can use [`textwrap.dedent`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) to remove the leading whitespaces

Reply:
thanks, but I don't think fauxfactory outputs spaces in alpha strings, it is just random ascii characters https://github.com/omaciel/fauxfactory/blob/c5d139ccb36d933f064576a1d0ea8832317a6f75/fauxfactory/factories/strings.py#L107